.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+Fixes #. A short summary which serves as a squashed-commit message.
+
+A full description of what's been implemented.
```

Comment:
I am not sure how `full description` is gonna be understood by people. I guess we don't expect a 500-word essay. Maybe `A description to understand introduced changes without reading the code`.

Reply:
Sounds better. I'm going to change to your version.
Anything else?